src/main/java/web/HomePage.java

```diff
@@ -0,0 +1,82 @@
+package web;
+
+import com.zebrunner.carina.webdriver.decorator.ExtendedWebElement;
+import com.zebrunner.carina.webdriver.decorator.PageOpeningStrategy;
+import com.zebrunner.carina.webdriver.gui.AbstractPage;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import org.openqa.selenium.By;
+import org.openqa.selenium.SearchContext;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.support.FindBy;
+import web.components.LoginModal;
+import web.components.NavBar;
+import web.components.ProductCard;
+import web.components.ProductCategory;
+
+public class HomePage extends AbstractPage {
+
+    @FindBy(xpath = "//button[@id='next2']")
+    private ExtendedWebElement nextButton;
+
+    @FindBy(xpath = "//button[@id='prev2']")
+    private ExtendedWebElement prevButton;
+
+    @FindBy(xpath = "//*[@class='card-block']")
+    private ExtendedWebElement cardLoadedMarker;
+
+    @FindBy(xpath = "//div[contains(@class, 'card ')]")
+    private List<ProductCard> productList;
+
+    @FindBy(xpath = "//a[@id = 'itemc']")
+    private List<ProductCategory> categories;
+
+    public HomePage(WebDriver driver) {
+        super(driver);
+        setPageOpeningStrategy(PageOpeningStrategy.BY_ELEMENT);
+        setUiLoadedMarker(cardLoadedMarker);
+    }
+
+    public void clickNextButton() {
+        nextButton.click();
+    }
+
+    public void clickPrevButton() {
+        prevButton.click();
+    }
```

Comment:
just to make it more understandable in the test
```suggestion
    public void clickNextPageButton() {
        nextPageButton.click();
    }

    public void clickPrevPageButton() {
        prevPageButton.click();
    }
```